CMakeLists.txt

```diff
@@ -88,10 +92,10 @@ if (CMAKE_USE_PTHREADS_INIT)
   # Common defines for most POSIX platforms.
   if (HOST MATCHES .*-.*-aix.*|.*-.*-android.*|.*-.*-cygwin.*|.*-.*-darwin.*|.*-.*-.*freebsd.*|.*-.*-haiku.*|.*-.*-gnu.*|.*-.*-hpux11.*|.*-.*-irix.*|.*-.*-.*linux.*|.*-.*-msys.*|.*-.*-nacl.*|.*-.*-netbsd.*|.*-.*-openbsd.*|.*-.*-osf.*|.*-.*-solaris.*)
     add_definitions("-DGC_THREADS -D_REENTRANT")
-    if (enable_parallel_mark)
```

Comment:
I'm going to leave lower case variable names for now (to match the configure)

Reply:
Got it.
Merged master into PR branch and used all lowercase names for options